miden-tx/tests/integration/wallet/mod.rs

```diff
@@ -69,17 +70,18 @@ fn prove_receive_asset_via_wallet() {
     let tx_script = executor
         .compile_tx_script(tx_script_code, vec![(target_pub_key, target_keypair_felt)], vec![])
         .unwrap();
+    let tx_args: TransactionArgs = TransactionArgs::new(Some(tx_script), None);
 
     // Execute the transaction and get the witness
-    let executed_transaction = executor
-        .execute_transaction(target_account.id(), block_ref, &note_ids, Some(tx_script))
+    let transaction_result = executor
```

Comment:
I don't think we need to change `executed_transaction` into `transaction_result` here and in other places in this file.